--- setup.py.req	2011-10-24 23:12:30.722600056 +0200
+++ setup.py	2011-10-24 23:13:28.430715437 +0200
@@ -284,7 +284,7 @@ else:
         'sqlalchemy >= 0.6',
         # buildbot depends on sqlalchemy internals, and these are the tested
         # versions.
-        'sqlalchemy-migrate ==0.6.0, ==0.6.1, ==0.7.0, ==0.7.1',
+        'sqlalchemy-migrate >= 0.6.0',
     ]
     # Python-2.6 and up includes json
     if not py_26:
